src/styles/components/_specification-block.scss

```diff
@@ -150,11 +151,6 @@
     }
   }
 
-  /* Adjacent sibling selector */
-  li + li {
-    margin-top: 1.5rem;
-  }
-
```

Comment:
This was causing there to be no space between the first nested bullet point and the one above it.